games-arcade/wormux/wormux-0~git20130927.recipe

```diff
@@ -0,0 +1,126 @@
+srcGitRev="e0dc503460055a14a0d7fd94e66d7484264ee62b"
+SOURCE_URI="https://github.com/a-team/wormux/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="wormux-$portVersion-$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d5e66808de2e4912a64b583e3be4b8ea52cd220b8617a113507cca2c2291c6d3"
+SOURCE_DIR="wormux-$srcGitRev"
+PATCHES="wormux-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/wormux directory keep-old
+#	$relativeSettingsDir/wormux/custom_team directory keep-old
+	"
+USER_SETTING_FILES="
+	data/wormux directory
+	"
+PROVIDES="
+	wormux$secondaryArchSuffix = $portVersion
+	app:Wormux = $portVersion
+	cmd:warmux = $portVersion
+	cmd:warmux_list_games = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libSDL_gfx$secondaryArchSuffix
+	lib:libSDL_image$secondaryArchSuffix
+	lib:libSDL_mixer_1.2$secondaryArchSuffix
+	lib:libSDL_net_1.2$secondaryArchSuffix
+	lib:libSDL_ttf_2.0$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
+#	lib:libSDL2$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcurl$secondaryArchSuffix
+#	devel:libfribidi$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libSDL_gfx$secondaryArchSuffix
+	devel:libSDL_image$secondaryArchSuffix
+	devel:libSDL_mixer_1.2$secondaryArchSuffix
+	devel:libSDL_net_1.2$secondaryArchSuffix
+	devel:libSDL_ttf_2.0$secondaryArchSuffix
+#	devel:libSDL_1.2$secondaryArchSuffix
+	devel:libSDL$secondaryArchSuffix
+#	devel:libSDL2$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libx11$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:libtoolize
+	"
+BUILD()
+{
+	mkdir -p $settingsDir/wormux
+	mkdir -p $settingsDir/wormux/custom_team
+	mkdir -p $dataDir/wormux
+	
+	export CFLAGS="-D_BSD_SOURCE"
+	libtoolize --force --copy --install
+	aclocal
+	autoconf
+	automake --add-missing
+#	autoreconf -vfi
+	runConfigure ./configure \
+		--with-font-path=$fontsDir/Ubuntu-R.ttf \
+		--with-datadir-name=$dataDir/wormux \
+		--with-localedir-name=$dataDir/wormux/locale \
+		--disable-dependency-tracking \
+		--build=x86
+#		--enable-servers #breaks
+#		--enable-fribidi # breaks
+	make $jobArgs
+}
+
+INSTALL()
+{
+#	mkdir -p $settingsDir
+#	mkdir -p $settingsDir/warmux
+	make install
+
+	# Cleanup
+	rm -rf $prefix/share #pixmap icons and .desktop files
+	mkdir -p $fontsDir
+	cp $dataDir/wormux/font/* $fontsDir
+	rm -rf $dataDir/wormux/data/font
+
+	# Link in appsDir and in menu
+	mkdir -p $appsDir
+	ln -s $prefix/bin/warmux $appsDir/Wormux
```

Comment:
If you want to install the binary in $appsDir you can omit bindir with configure and use "--bindir=$appsDir" as an option there, this would make this line obsolete